config/application.rb

```diff
@@ -31,7 +31,7 @@ class Application < Rails::Application
     config.email_url =
       if Rails.env.production?
         "https://www.fileyourstatetaxes.org"
-      elsif Rails.env.development?
+      elsif Rails.env.development? || Rails.env.staging?
         "https://demo.fileyourstatetaxes.org"
       else
         "http://localhost:3000"
```

Comment:
I suggest below:

```
  if ENV["REVIEW_APP"] == "true"
      "https://demo.fileyourstatetaxes.org"
  elsif Rails.env.production?
      "https://www.fileyourstatetaxes.org"
  else
     "http://localhost:3000"
  end
```

- heroku/staging has env of `production` so we dont want to evaluate production first
- Rails development should probably use `localhost` unless there's a reason to use demo as the email url locally